csrc/scheduler/transpose.cpp

```diff
@@ -127,6 +127,27 @@ void TransposeScheduler::computeHeuristics(
 
 namespace {
 
+// propagation could miss trivial reduction iterdomain on input tensors, since
```

Comment:
As Xiang also commented, trivial reduction could be confusing as it would imply squeeze. Can you please rephrase the comment?